tests/integration/test_install_basic.py

```diff
@@ -556,3 +556,20 @@ def test_install_tarball_is_actually_installed(pipenv_instance_pypi):
         assert c.returncode == 0
         c = p.pipenv("""run python -c "from dataclasses_json import dataclass_json" """)
         assert c.returncode == 0
+
+@pytest.mark.basic
+@pytest.mark.install
+def test_packages_sorted_alphabetically_in_category(pipenv_instance_private_pypi):
+    with pipenv_instance_private_pypi() as p:
+        with open(p.pipfile_path, "w") as f:
+            contents = """
+[packages]
+atomicwrites = "*"
+colorama = "*"
+            """.strip()
+            f.write(contents)
+        c = p.pipenv("install build")
+        assert c.returncode == 0
+        assert "build" in p.pipfile["packages"]
+        assert list(p.pipfile["packages"].keys()) == ["atomicwrites", "build", "colorama"]
+
```

Comment:
For this test I just chose three arbitrary packages that are already used inside pipenv's own Pipfile.